consensus/pacemaker_consensus.go

```diff
@@ -72,14 +72,17 @@ func (m *consensusModule) NewLeader(msg *anypb.Any) error {
 	return m.electNextLeader(message)
 }
 
-func (m *consensusModule) GetPrepareQC() *anypb.Any {
+func (m *consensusModule) GetPrepareQC() (*anypb.Any, error) {
+	//TODO! check why testing fails when prepareQC is nil
```

Comment:
Good opportunity for a unit test!

Reply:
I am currently writing an initial test for state sync server #409 in order to see if it works. What about basing this branch to the main once #404 is merged, and writing the unit tests in the new framework?

Reply:
After a review, this is normal to have prepareQC nil, so checking if its nil and returning an error is not correct. Therefore, test is not required here, imo.